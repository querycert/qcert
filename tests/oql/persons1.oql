/* Simple select-from-where */
select p
from p in Customers
where p->age = 32
