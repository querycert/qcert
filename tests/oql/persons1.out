[{"$class": ["Customer"], "$data": {"age": { "$nat" : 32 }, "cid": { "$nat" : 123 }, "name": "John Doe"}}, {"$class": ["Customer"], "$data": {"age": { "$nat" : 32 }, "cid": { "$nat" : 124 }, "name": "Jane Doe"}}, {"$class": ["Customer"], "$data": {"age": { "$nat" : 32 }, "cid": { "$nat" : 126 }, "name": "Jill Does"}}]